projects/fal/src/fal/cli.py

```diff
@@ -257,7 +257,7 @@ def load_function_from(
         raise api.FalServerlessError(f"Function '{function_name}' not found in module")
 
     target = module[function_name]
-    if issubclass(target, fal.App):
+    if isinstance(target, type) and issubclass(target, fal.App):
```

Comment:
What is `type` in this scope?

Reply:
all classes (types) are an instance of type (type the built-in object type, which is the metaclass of all classes).